Doc/library/functions.rst

```diff
@@ -1826,6 +1826,10 @@ are always available.  They are listed here in alphabetical order.
       Negative values for *level* are no longer supported (which also changes
       the default value to 0).
 
+   .. versionchanged:: 3.9
+      When the command line options :option:`-E` or :option:`-I` is used,
```

Comment:
```suggestion
      When the command line options :option:`-E` or :option:`-I` are used,
```

Reply:
Will fix both comments now

Reply:
I understood what the problem was, apparently I misconfigured my local repo so there was no up stream, now everything should pass fine.